frontend/src/index.jade

```diff
@@ -100,19 +106,16 @@ html
             label
               input(type="checkbox", v-model="filterSortReverse").mui-checkbox.summary-picker__checkbox
               span reverse
-            label
-              input(type="checkbox", v-model="filterGroupRepos").mui-checkbox.summary-picker__checkbox
-              span group by repo/branch
         #summary-charts
-          .summary-charts(v-for="repo of filteredRepos")
-            .summary-charts__title(
-              v-if="filterGroupRepos",
-            ) {{ repo[0].repoPath }}
+          .summary-charts(v-for="repo of filteredRepos", v-if="repo.length>0")
+            .summary-charts__title(v-if="filterGroupSelection === 'groupByRepos'") {{ repo[0].repoPath }}
+            .summary-charts__title(v-if="filterGroupSelection === 'groupByAuthors'") {{ repo[0].displayName }}
             .summary-chart(v-for="(user, i) in repo")
               .summary-chart__title
                 .summary-chart__title--index {{ i+1 }}
-                .summary-chart__title--repo(v-if="!filterGroupRepos") {{ user.repoPath }}
-                .summary-chart__title--name {{ user.displayName }}
+                .summary-chart__title--repo(v-if="filterGroupSelection === 'groupByNone'") {{ user.repoPath }}
+                .summary-chart__title--author-repo(v-if="filterGroupSelection === 'groupByAuthors'") {{ user.repoPath }}
+                .summary-chart__title--name(v-if="filterGroupSelection !== 'groupByAuthors'") {{ user.displayName }}
```

Comment:
Shouldn't this be `filterGroupSelection === 'groupByRepo'`?

Reply:
The user's display name has to be shown when grouped `by none` and `by repo`.